code/game/objects/items/explosives/mine.dm

```diff
@@ -34,6 +37,9 @@
 	if(tripwire)
 		qdel(tripwire)
 		tripwire = null
+	if(camera)
+		qdel(camera)
+		camera = null
```

Comment:
`QDEL_NULL`

Reply:
QDEL_NULL combines both steps into one, you're now trying to null something that doesn't exist.